libs/utils/executor.py

```diff
@@ -272,13 +272,29 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        errors = False
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
+            return True
+        for name in tc['files']:
+            logging.info('%14s - Write value into file: \'%s\' -> \'%s\'',
+                         'Files', tc['files'][name], name)
```

Comment:
It would be clearer to use `for name, value in tc['files'].iteritems()` above then use `value` instead of `tc['files'][name]` here and in the `write_value` call.